OWNERS_ALIASES

```diff
@@ -112,6 +112,28 @@ aliases:
     - ClaudiaJKang
     - gochist
     - ianychoi
+  sig-docs-l10n-owners: # Owners for localization projects
+    - alexbrand # Spanish
+    - bene2k1 # German
+    - bradamant3 # English
+    - chenrui # Chinese
+    - claudiajkang # Korean
```

Comment:
```suggestion
    - ClaudiaJKang # Korean
```

- My correct Github ID is `ClaudiaJKang`. :D 
  Could please you change this line?

Reply:
DOne :D